etc/HIProdOfflineConfiguration.py

```diff
@@ -1554,6 +1580,41 @@
                dqm_sequences=["@commonSiStripZeroBias", "@ecal", "@hcal", "@muon", "@jetmet", "@egamma"],
                physics_skims=["PbPbEMu", "PbPbZEE", "PbPbZMM", "LogError", "LogErrorMonitor"],
                scenario=hiRawPrimeScenario)
+    
+DATASETS = ["HIPhysicsRawPrime0"]
+
+for dataset in DATASETS:
+    addDataset(tier0Config, dataset,
+               do_reco=True,
+               raw_to_disk=False,
+               aod_to_disk=False,
+               #timePerEvent=7,
+               write_dqm=False,
+               disk_node="T2_US_Vanderbilt",
+               alca_producers=["EcalUncalZElectron", "EcalUncalWElectron", "EcalESAlign", "MuAlCalIsolatedMu", 
+                               "TkAlDiMuonAndVertex", "HcalCalHO", "HcalCalIsoTrkProducerFilter", "HcalCalHBHEMuonProducerFilter",
+                               "SiStripCalZeroBias", "SiStripCalMinBias","TkAlMinBias",
+                               "HcalCalIsolatedBunchSelector", "HcalCalIterativePhiSym","HcalCalMinBias",
+                               "TkAlJpsiMuMu", "TkAlUpsilonMuMu","TkAlZMuMu","TkAlMuonIsolated"],
+               dqm_sequences=["@none"],
+               physics_skims=["PbPbEMu", "PbPbZEE", "PbPbZMM", "LogError", "LogErrorMonitor"],
+               scenario=hiRawPrimeScenario)
```

Comment:
This doesn't look right. The DQM sequences are different than what we have now 
![image](https://github.com/dmwm/T0/assets/57130402/482e25b2-811e-4cc4-996c-1f7457329d61)

Reply:
I would rather remove all this, and just modify 
![image](https://github.com/dmwm/T0/assets/57130402/35ccfa3c-275c-4cbf-ab27-d78d2e5b5e54)
  `DATASETS += ["HIPhysicsRawPrime0"]`. HIPhysicsRawPrime1 and other look okay.